hdr_html_canvas_element.md

```diff
@@ -121,101 +122,36 @@ For WebGPU this would be:
+* Canvas 2D's proposed [Canvas Floating Point Color Values](https://github.com/w3c/ColorWeb-CG/blob/main/canvas_float.md) allows for
+  specifying higher bit depth canvas and ImageData.
+* WebGL's proposed [``drawingBufferStorage``](https://github.com/KhronosGroup/WebGL/pull/3222) function allows for specifying higher
+  bit depth formats.
 
-WebGPU's ``GPUCanvasConfiguration`` can allow for specifying higher bit depth formats.
+[The WebGPU ``GPUCanvasConfiguration``](https://www.w3.org/TR/webgpu/#canvas-configuration) already allows 16-bit float values and
+can be further extended if needs be.
 
-### Display of floating point color values
+### Extend the existing `srgb` color space for HDR imagery
 
-Define a screen's native linear color space to be the color space with color primaries and white point set to the screen's `ScreenDetailed`'s `colorVolume` and the identity as its transfer function.
+Define a *screen's native linear color space* to be the color space with color primaries and white point set to the screen's
+`ScreenDetailed`'s `colorVolume` and the identity as its transfer function.
 
-A color is said to be within a screen's default range if, when that color is converted to the screen's native linear color space (using relative colorimetric intent), all of its color components are within the `[0, 1]` interval.
-When displaying content produced by a rendering context with `CanvasHighDynamicRangeMode` set to `'default'`, all colors that are within the screen's default range must be displayed without any clamping or roll-offs (neither in chrominance nor luminance).
+A color is said to be within a screen's *default* range if, when that color is converted to the screen's native linear color space
+(using relative colorimetric intent), all of its color components are within the `[0, 1]` interval. When displaying content produced
+by a rendering context with `CanvasHighDynamicRangeMode` set to `'default'`, all colors that are within the screen's default range
+must be displayed without any clamping or roll-offs (neither in chrominance nor luminance).
 
-A color is said to be within a screen's extended range if, when that color is converted to the screen's native linear color space (using relative colorimetric intent), all of its color components are within the `[0, highDynamicRangeHeadroom]` interval.
-When displaying content produced by a rendering context with `CanvasHighDynamicRangeMode` set to `'extended'`, all colors that are within the screen's extended range must be displayed without any clamping or roll-offs (neither in chrominance nor luminance).
+A color is said to be within a screen's *extended* range if, when that color is converted to the screen's native linear color space
+(using relative colorimetric intent), all of its color components are within the `[0, highDynamicRangeHeadroom]` interval. When
```

Comment:
That definition carefully excludes out of gamut colors, which would have some negative values, while leaving in-gamut HDR values. Nice.